eslzArm/eslzArm.json

```diff
@@ -1494,6 +1496,26 @@
                 }
             }
         },
+        {
+            // Assigning Cost Optimization policy initiative to intermediate root management group
+            //"condition": "[and(or(not(empty(parameters('managementSubscriptionId'))), not(empty(parameters('singlePlatformSubscriptionId')))), equals(parameters('enableAsc'), 'Yes'), equals(environment().resourceManager, 'https://management.azure.com/'))]",
```

Comment:
Why is this commented out? Remove the 'equals(parameters('enableAsc'), 'Yes')' requirement, but the rest is valid. Should not comment out code in JSON.

Reply:
@springstone this is still here????